### PR TITLE
feat: wxt

### PR DESCRIPTION
**Pull request by 42Willow**
note: there's duplicated code at the moment we need to figure out how to get rid of!! @BC-548 I need help here :|
<del>Vite maybe? We need some sort of compile step.</del> yep , waiting on #63 

- [x] get rid of duplicated code

Pretty much refactored the entire extension to remove these 100 lines of duplicated code... it better be worth it!

### CODE REVIEW

**Comment by 42Willow** [comment]
I think we'll use vite to fix this duplicated code issue- It also integrates with tailwind

**Comment by 42Willow** [comment]
@BC-548 may as well merge now and develop on main, since this has all the functionality of main now.

File: package.json [hunk 1]
```diff
@@ -1,18 +1,40 @@
...
+    "pino": "^9.2.0",
+    "postcss": "^8.4.38",
+    "prettier": "^3.3.2",
+    "prettier-plugin-svelte": "^3.2.4",
+    "prettier-plugin-tailwindcss": "^0.5.14",
+    "svelte": "^4.2.18",
+    "svelte-check": "^3.8.0",
+    "svelte-spa-router": "^4.0.1",
+    "tailwindcss": "^3.4.4",
+    "tslib": "^2.6.3",
```
Comment (42Willow) [inline package.json:20]: what's `tslib` for

File: src/.web-extension-id
Comment (42Willow) [inline src/.web-extension-id:1]: related to #64

File: src/utils/storage.ts [hunk 2]
```diff
@@ -0,0 +1,141 @@
+// const defaultSettings = {
```
Comment (42Willow) [inline src/utils/storage.ts:1]: might remove this comment

**Review by BC-548 — APPROVED** [review]
That was a lot of reading!

**Pull request merged**